core/vm/eips.go

```diff
@@ -339,12 +339,10 @@ func opExtCodeCopyEIP4762(pc *uint64, interpreter *EVMInterpreter, scope *ScopeC
 	addr := common.Address(a.Bytes20())
 	code := interpreter.evm.StateDB.GetCode(addr)
 	paddedCodeCopy, copyOffset, nonPaddedCopyLength := getDataAndAdjustedBounds(code, uint64CodeOffset, length.Uint64())
-	if !scope.Contract.IsSystemCall {
```

Comment:
Note that this implements the suggestion that extcodecopy of a "system contract" (yeah I know you guys don't like the name but it has the advantage that everyone gets what I'm talking about) will end up in the witness. Spec change is underway, I'll hack the stuff back in when I sync the testnet in the meantime.